src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyAmendmentService.cs

```diff
@@ -18,24 +18,47 @@ public class MethodologyAmendmentService : IMethodologyAmendmentService
         private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
         private readonly IUserService _userService;
         private readonly IMapper _mapper;
+        private readonly ContentDbContext _context;
 
         public MethodologyAmendmentService(
             IPersistenceHelper<ContentDbContext> persistenceHelper, 
             IUserService userService,
-            IMapper mapper)
+            IMapper mapper, 
+            ContentDbContext context)
         {
             _persistenceHelper = persistenceHelper;
             _userService = userService;
             _mapper = mapper;
+            _context = context;
         }
 
         public async Task<Either<ActionResult, MethodologySummaryViewModel>> CreateMethodologyAmendment(
             Guid originalMethodologyId)
         {
             return await _persistenceHelper.CheckEntityExists<Methodology>(originalMethodologyId)
                 .OnSuccess(_userService.CheckCanMakeAmendmentOfMethodology)
-                .OnSuccessDo(() => throw new NotImplementedException())
+                .OnSuccess(HydrateMethodologyForAmendment)
+                .OnSuccess(CreateAndSaveAmendment)
                 .OnSuccess(_mapper.Map<MethodologySummaryViewModel>);
```

Comment:
I figured that after adding the Methodology image code this class might grow larger, but possibly this could be rolled into the MethodologyService, yeah.  I could also wire in MethodologyService for the time being and call GetSummary.

I figured a separate service for Amendment might be worthwhile as it tends to end up being an area of complexity and so maybe having it as a separate area of code that's self-contained might be worth having, rather than it all getting lost in a much bigger service.

Reply:
For the time being, I've wired in MethodologyService and called GetSummary.